packages/polywrap-msgpack/polywrap_msgpack/__init__.py

```diff
@@ -52,10 +52,12 @@ def sanitize(value: Any) -> Any:
     """
     if isinstance(value, dict):
         dictionary: Dict[Any, Any] = value
-        for key, val in dictionary.items():
+        for key, val in list(dictionary.items()):
             if isinstance(key, str):
                 dictionary[key] = sanitize(val)
-            else:
+            elif key.uri:
```

Comment:
I'm not familiar with this codebase, but why do we want to check whether the `key.uri` exists here? This msgpack package should be independent of any of the polywrap specific semantics, so referencing "uri" doesn't quite make sense here to me.

Reply:
It shouldn't be done like this, I will update it.